stdlib/REPL/src/precompile.jl

```diff
@@ -0,0 +1,201 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+module Precompile
+# Can't use this during incremental: `@eval Module() begin``
+
+import ..REPL
+
+Base.include(@__MODULE__, joinpath(Sys.BINDIR, "..", "share", "julia", "test", "testhelpers", "FakePTYs.jl"))
```

Comment:
This is a duplication of `contrib/generate_precompile.jl` a bit simplified,
but it is awkward to duplicate the functionality.